docs/module/srv6.md

```diff
@@ -1,18 +1,61 @@
 # Segment Routing over IPv6 (srv6) Configuration Module
 
-This configuration module configures SRv6 over IS-IS and (optionally) BGP on Nokia SR OS.
+This configuration module configures SRv6 over IS-IS and (optionally) BGP and L3VPN
 
 Supported SRv6 features:
 
 * End SIDs (nodes) and End-x SIDs (links), all statically allocated
 * IS-IS routing protocol with SRv6 TLVs
 * Dynamic BGP Service SIDs
+* BGP L3VPN v4/v6 over SRv6
 
 The module currently depends on IS-IS module and will trigger a configuration error if the **isis** module is not enabled in the network topology.
 
-## Parameters
+## Platform Support
+The following table describes per-platform support of individual router-level SRv6 features:
 
-* Locator: an IPv6 address range to allocate to SRv6 nodes; each node is assigned a unique /64 prefix from this range
+| Operating system         | IS-IS | OSPFv3 | BGP v4/v6 |
+| ------------------------ |:-----:|:------:|:---------:|
+| FRR                      |   ✅  ||  ❌   |     ✅    |
+| Nokia SR OS              |   ✅  ||  ❌   |     ✅    |
+
+(srv6-l3vpn-supported-platforms)=
+### BGP/SRv6 L3VPN
+
+| Operating system      | VPNv4 | VPNv6 |
+| ----------------------| :---: | :---: |
+| FRR                   |   ✅  |   ✅  |
+| Nokia SR OS           |   ❌  |   ❌  |
+
+**Notes**
+* VPNv4 and VPNv6 address families are enabled on IPv6 IBGP/EBGP sessions
+
+## Configurable Global and Node Parameters
+
+* **addressing.srv6_locator** -- global address pool for allocation of SRv6 locator prefixes, default prefix defined by `topology.defaults.srv6.locator_pool`
```

Comment:
Any reason why you won't say what it is?

Reply:
it's a surprise ;)